memory/src/main/scala/filodb.memory/format/BinaryVector.scala

```diff
@@ -448,6 +486,11 @@ object PrimitiveVectorReader {
   final def bitShift(vector: BinaryVectorPtr): Int = UnsafeUtils.getByte(vector + OffsetBitShift) & 0x03f
   final def signed(vector: BinaryVectorPtr): Boolean =
     (UnsafeUtils.getByte(vector + OffsetNBits) & SignMask) != 0
+  final def resetted(vector: BinaryVectorPtr): Boolean =
+    (UnsafeUtils.getShort(vector + OffsetNBits) & ResetMask) != 0
+
+  final def markReset(vector: BinaryVectorPtr): Unit =
```

Comment:
Naming this markReset kept tripping me up. Java streams and Buffers have a special use for "mark" and "reset". These are both verbs. How about setCounterReset and hasCounterReset? Not a big deal either way.